config/settings.js

```diff
@@ -27,11 +27,20 @@
  */
 
 window.spinnakerSettings = {
+  defaultProviders: ['aws', 'gce', 'azure', 'cf', 'kubernetes'],
+  feedbackUrl: process.env.FEEDBACK_URL,
   gateUrl: gateUrl,
-  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true',
+  bakeryDetailUrl: bakeryBaseUrl + '/api/v1/global/logs/{{context.status.id}}?html=true'
+  authEndpoint: process.env.AUTH_ENDPOINT,
   pollSchedule: 30000,
-  defaultTimeZone: 'America/New_York', // see http://momentjs.com/timezone/docs/#/data-utilities/
+  defaultTimeZone: process.env.TIMEZONE || 'America/Los_Angeles', // see http://momentjs.com/timezone/docs/#/data-utilities/
   providers: {
+    azure: {
+      defaults: {
+        account: 'azure-test',
```

Comment:
@AMeng 
If you want to bind a variable to a spring configuration value (from spinnaker-local.yml + spinnaker.yml + /etc/default/spinnaker) then declare a var comment for it as you did, then reference that variable.